concourse/enumerator.py

```diff
@@ -235,7 +256,11 @@ def _scan_repository_for_definitions(
 
             verbose('from repo: ' + repository.name + ':' + branch_name)
             try:
-                definitions = load_yaml(definitions.decoded.decode('utf-8'))
+                decoded_definitions = definitions.decoded.decode('utf-8')
+                info(f'Linting pipeline_definitions for {repository}')
+                lint_yaml(decoded_definitions, config=REPLICATION_LINTING_CFG)
+                definitions = load_yaml(decoded_definitions)
```

Comment:
how about adding a `lint` (+lint-cfg) flag to `load_yaml`?

Reply:
done.
Not _strictly_ part of this PR, however.